opendoors/open-doors-tables-working.sql

```diff
@@ -164,6 +164,7 @@ CREATE TABLE IF NOT EXISTS `tags`
     `ao3_tag_type`         varchar(255)  DEFAULT NULL,
     `ao3_tag_category`     varchar(255)  DEFAULT NULL,
     `ao3_tag_fandom`       varchar(255)  DEFAULT NULL,
+	`storyid`			   int(11)		 DEFAULT NULL,
```

Comment:
We don't need to add `storyid` to this version of the `tags` table because it now uses join tables to match tags to the stories or story_links they're associated with. The ODAP used to use a massive denormalised `tags` table which was always a stupid idea and very buggy! 
Changes are needed in the ODAP, though, which I've put in OD-574.